app-editors/liteide/autobuild/build

```diff
@@ -0,0 +1,31 @@
+pushd build
```

Comment:
Please lint this script and mark `abinfo`, use verbosity, etc. in accordance with our [Package Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/).